packages/components/scripts/vendorism/custom/details.vendorism.js

```diff
@@ -29,39 +40,36 @@ const transformComponent = (path, originalContent) => {
     ['\'details--rtl', 'isRtl'],
   ], content);
 
-  // Add support for size attribute
-  content = addSectionAfter(
-    content,
-    '@property({ type: Boolean, reflect: true }) disabled = false;',
-    `
-  /** The details's size. */
-  @property({ reflect: true }) size: 'medium' | 'large' = 'medium';
-    `,
-  );
-
-  content = addSectionAfter(
-    content,
-    'details: true,',
-    `          'details--size-medium': this.size === 'medium',
-          'details--size-large': this.size === 'large',
-    `,
-  );
-
-  // Add support for contained attribute
-  content = addSectionAfter(
-    content,
-    '@property({ type: Boolean, reflect: true }) disabled = false;',
-    `
-  /** Draws the details as contained element. */
-  @property({ type: Boolean, reflect: true }) contained = false;
-    `,
-  );
+  content = addSectionsAfter([
+    // Add support for size attribute
+    [
+      '@property({ type: Boolean, reflect: true }) disabled = false;',
+      `/** The details's size. */
+  @property({ reflect: true }) size: 'medium' | 'large' = 'medium';`,
+      { newlinesBeforeInsertion: 2, tabsBeforeInsertion: 1 },
+    ],
+    // Add classes for size attribute
+    [
+      'details: true,',
+      `'details--size-medium': this.size === 'medium',
+          'details--size-large': this.size === 'large',`,
+      { tabsBeforeInsertion: 5 },
+    ],
 
-  content = addSectionAfter(
-    content,
-    '\'details--disabled\': this.disabled,',
-    '          \'details--contained\': this.contained,',
-  );
+    // Add support for contained attribute
+    [
+      '@property({ type: Boolean, reflect: true }) disabled = false;',
+      `/** Draws the details as contained element. */
+  @property({ type: Boolean, reflect: true }) contained = false;`,
+      { newlinesBeforeInsertion: 2, tabsBeforeInsertion: 1 },
+    ],
+    // Add classes for 'contained' attribute
+    [
+      '\'details--disabled\': this.disabled,',
+      '\'details--contained\': this.contained,',
+      { tabsBeforeInsertion: 5 },
```

Comment:
Same for here. Formatting looks strange in git diffs as the tab-width may be set incorrectly. See attached image for example.

<img width="688" alt="Bildschirmfoto 2024-07-24 um 07 36 46" src="https://github.com/user-attachments/assets/e1c6a12b-af3b-4cdb-9a16-1cb8276f3d3c">

Reply:
I changed adding a tab with adding two spaces for the addSectionAfter and the addSectionBefore methods